src/Turtle/Prelude.hs

```diff
@@ -1725,6 +1725,15 @@ inplaceWith sed_ pattern' file = liftIO (runManaged (do
     mv tmpfile file ))
 
 -- | Search a directory recursively for all files matching the given `Pattern`
+-- Note: The `Pattern` matches against the full path of a file as opposed to
+-- just the base name as in GNU find. For example:
+--
+-- > find "foo.txt" "./bar" 
+--
+-- will return only a single filepath: @./bar/foo.txt@. To search for a
```

Comment:
Are you sure this is correct?  Wouldn't you need to search for the exact string `"./bar/foo.txt"` to match that path?

Reply:
Oh your right! It was originally `find "foo.txt" "."` and then I went back and updated.